dev-compose.yaml

```diff
@@ -0,0 +1,93 @@
+name: photoview
+
+services:
+  mysql:
+    image: mariadb:lts
+    environment:
+      MYSQL_DATABASE: photoview_test
+      MYSQL_USER: photoview
+      MYSQL_PASSWORD: photosecret
+      MYSQL_RANDOM_ROOT_PASSWORD: yes
+    healthcheck:
+      test:
+      - CMD
+      - mariadb-admin
+      - --user=photoview
+      - --password=photosecret
+      - ping
+      interval: 20s
+      timeout: 5s
+      retries: 10
+    volumes:
+    - ./dev/mysql:/var/lib/mysql:rw
+    ports:
+    - 3306:3306
+
+  postgres:
+    image: postgres:16-alpine
+    environment:
+      POSTGRES_DB: photoview_test
+      POSTGRES_USER: photoview
+      POSTGRES_PASSWORD: photosecret
+    healthcheck:
+      test:
+      - CMD
+      - pg_isready
+      interval: 20s
+      timeout: 5s
+      retries: 10
+    volumes:
+    - ./dev/postgres:/var/lib/postgresql/data:rw
+    ports:
+    - 5432:5432
+
+  api-test:
```

Comment:
You can check `scripts/test_api.sh` on how to use this service.

It's because env files differ a lot between developing and testing. To pass tests, `PHOTOVIEW_API_ENDPOINT` must be an empty string.

I hope to update tests not depending on env variables later, but not this PR.